src/components/Provider.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Provider(props) {
```

Comment:
Ik weet niet precies waar deze providercomponent voor is, misschien ben je vergeten deze te verwijderen.

Reply:
Klopt, hoorde nog bij mijn eerste opzet.